lib/openxml/has_properties.rb

```diff
@@ -18,8 +18,9 @@ def value_property(name, as: nil, klass: nil)
         attr_reader name
 
         properties[name] = (as || name).to_s
+        classified_name = properties[name].split("_").map(&:capitalize).join
         class_name = klass.name unless klass.nil?
-        class_name ||= "Properties::#{properties[name].split("_").map(&:capitalize).join}"
+        class_name ||= (inspect.split("::")[0...-2] + ["Properties", classified_name]).join("::")
```

Comment:
You use `to_s` when you did this for attributes. I wonder if that might be better than `inspect`? — since they're for different purposes?

Reply:
Made the change 👍